e2e/helper/faultUtils.js

```diff
@@ -58,8 +58,15 @@ async function navigateToFaultManagementWithoutExample(page) {
 async function navigateToFaultItemInTree(page) {
     await page.goto('./', { waitUntil: 'networkidle' });
 
-    // Click text=Fault Management
-    await page.click('text=Fault Management'); // this verifies the plugin has been added
+    const mainTree = page.getByRole('tree', {
```

Comment:
can we just chain the selectors at this point?

Reply:
Probably. I think this example has been copied around and was originally used somewhere where we reused the `mainTree` locator.